sbol2/property.py

```diff
@@ -597,7 +597,7 @@ def find_resource(self, uri, resource_namespaces, object_store,
                 if obj.identity == compliant_uri:
                     return obj
 
-    def get(self, uri):
+    def get(self, uri = ''):
```

Comment:
This probably generates a pycodestyle warning. Remove the spaces around the equal sign to fix that up:
```suggestion
    def get(self, uri=''):
```

Reply:
Fixed in 6a9f2b9dde1741b04d76f08ee051cc7a13fa7c86